app/javascript/__tests__/data/listing-rental-closed.ts

```diff
@@ -0,0 +1,301 @@
+import RailsRentalListing from "../../api/types/rails/listings/RailsRentalListing"
+
+export const closedRentalListing: RailsRentalListing = {
+  unitSummaries: {
+    reserved: null,
+    general: [
+      {
+        unitType: "1 BR",
+        totalUnits: 1,
+        minSquareFt: null,
+        minRentalMinIncome: 5255,
+        // "minPriceWithParking": null,
+        // "minPriceWithoutParking": null,
+        minPercentIncome: null,
```

Comment:
I copied these test data files from api responses on full. Ts is mad because not all of these fields are defined in the type. Any thoughts on adding these all to the appropriate types or leaving these commented out? Some of the offenders are odd e.g. the `listing-sale-open.ts` response has `minRentalMinIncome`. The other option is asking whether we can suppress inappropriate fields on the Salesforce side

Reply:
If I'm understanding correctly I like the idea of adding these to the type since I think we will need them on the type eventually so that we can be confident the API response is what we expect it to be